apps/server/src/modules/authentication/controller/dto/account-search.response.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Account } from '@shared/domain';
+
+export class AccountSearchResponse {
+	constructor(account: Account) {
```

Comment:
As far as I know this is not forbidden by our architecture guidelines.
But generally the response DTO and the database entity should not know each other.